src/screens/CandidateDetail/TabBar/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import AboutTab from './AboutTab';
+import MatchTab from './MatchTab';
+import NewsTab from './NewsTab';
+import TabItem from './TabItem';
+
+class TabBar extends Component {
+  state = {
+    selectedTab: 'Match',
+  }
+
+  // Replace with Redux Action
```

Comment:
Why do you think this requires a redux action? Does any other part of the app care what tab you're looking at?

Reply:
Hmm. Great question! I suppose it doesn’t need one! I think my logic was that it would save a global reference to the current tab so if you navigated away or changed candidates it would drop you back at the tab you were most interested in.